stubtest.allowlist.to_review

```diff
@@ -24,3 +24,5 @@ PyQt5.QtTest.QAbstractItemModelTester.FailureReportingMode.__new__
 PyQt5.QtWebEngineCore.QWebEngineUrlScheme.Syntax.__new__
 PyQt5.QtWebEngineWidgets.QWebEnginePage.LifecycleState.__new__
 PyQt5.QtWidgets.QActionGroup.ExclusionPolicy.__new__
+PyQt5.QtCore.pyqtProperty
+PyQt5.QtQml.QQmlListProperty
```

Comment:
We just don't use these anywhere, so skipping for now.  Not sure why they show up now and not before.

Reply:
I am working on adding pyqtProperty . Not sure why they show up either...

Reply:
They should show up, I think.  The stub test is checking to make sure we have each thing documented properly in the `.pyi` files.  We don't for these.  But, I don't think we care for the sake of this dependency update PR since it isn't actually indicating any regression.  Certainly, having them in place would be good.  It seems like there will be lots of caveats for it.

Reply:
There were probably some fixes in stubtest.py which prevented these missing methods to show up.